editor/src/components/navigator/navigator.tsx

```diff
@@ -148,28 +176,18 @@ export const NavigatorComponent = betterReactMemo('NavigatorComponent', () => {
       onContextMenu={onContextMenu}
       id={NavigatorContainerId}
       tabIndex={-1}
+      style={{ height: '100%' }}
     >
       <SectionTitleRow minimised={minimised} toggleMinimised={toggleTwirler}>
         <FlexRow flexGrow={1}>
           <Title>Elements</Title>
         </FlexRow>
       </SectionTitleRow>
-      <SectionBodyArea minimised={minimised}>
+      <SectionBodyArea minimised={minimised} flexGrow={1}>
         <ElementContextMenu contextMenuInstance={'context-menu-navigator'} />
-        {navigatorTargets.map((targetPath, index) => {
-          const componentKey = TP.toComponentId(targetPath)
-          return (
-            <NavigatorItemWrapper
-              key={componentKey}
-              index={index}
-              targetComponentKey={componentKey}
-              templatePath={targetPath}
-              getMaximumDistance={getDistanceFromAncestorWhereImTheLastLeaf}
-              getDragSelections={getDragSelections}
-              getSelectedViewsInRange={getSelectedViewsInRange}
-            />
-          )
-        })}
+        <div style={{ flex: '1 1 auto' }}>
+          <AutoSizerComponent disableWidth={true}>{ItemList}</AutoSizerComponent>
```

Comment:
that's a weird kind of HOC! 

I wonder if we support this style on the canvas?

https://utopia.pizza/p/24153547-perpetual-action/

wow we do! :D

Reply:
Yeah, they expect a certain parameter in their case rather than having an arity of 0.